IO 1.5.0, Util 1.5.0

eed3si9n opened the pull request.

eed3si9n committed:
IO 1.5.0, Util 1.5.0

typesafe-tools commented:
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=pull/971/head&io=develop&librarymanagement=develop&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

eed3si9n committed:
Skip console test on Windows

Problem
-------

The following test is failing on AppVeyor (Windows):

```scala
  test("A compiler bridge should run console") {
    withTemporaryDirectory { tempDir =>
      console(tempDir.toPath)(":q")
    }
  }
```

https://ci.appveyor.com/project/sbt/zinc/builds/38543349

```
Welcome to Scala 2.12.13 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_162).
Type in expressions for evaluation. Or try :help.
scala> <console>:1: error: illegal start of definition
:q
^
```

Solution
--------
For now skip this test on Windows.
